pkg/eval/math/math.go

```diff
@@ -0,0 +1,286 @@
+// Package math exposes functionality from Go's math package as an elvish
+// module.
+package math
+
+import (
+	"math"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var e
+//
+// ```elvish
+// $math:e
+// ```
+//
+// The value of E: 2.718281.... This variable is read-only.
+
+//elvdoc:var pi
+//
+// ```elvish
+// $math:pi
+// ```
+//
+// The value of Pi: 3.141592.... This variable is read-only.
+
+//elvdoc:fn abs
+//
+// ```elvish
+// math:abs $number
+// ```
+//
+// Compute the absolute value of a number. Example:
+//
+// ```elvish-transcript
+// ~> math:abs 1.2
+// ▶ (float64 1.2)
+// ~> math:abs -5.3
+// ▶ (float64 5.3)
+// ```
+
+//elvdoc:fn ceil
+//
+// ```elvish
+// math:ceil $number
+// ```
+//
+// Compute the floor value of a number. Example:
```

Comment:
This should say "ceiling".

Also, I like how https://godoc.org/math#Ceil actually explains the definition of ceiling and lists the special cases.

Reply:
Yes, stupid copy/paste mistake. I will add a "Read the Go documentation..." link but that begs the question of when such links should be present.